README.md

```diff
@@ -1,33 +1,36 @@
 # Home Assistant Add-on: frigate
 
-An Home Assitant Add-on for Frigate - Realtime Object Detection for IP Cameras  
+An Home Assistant Add-on for Frigate - Realtime Object Detection for IP Cameras
 All the heavy lifting is done by https://github.com/blakeblackshear/frigate
-Uses OpenCV and Tensorflow to perform realtime object detection locally for IP cameras. Designed for integration with HomeAssistant or others via MQTT.
+Uses OpenCV and Tensorflow to perform realtime object detection locally for IP
+cameras. Designed for integration with HomeAssistant or others via MQTT.
 
-![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+![Supports aarch64 Architecture][aarch64-shield]
+![Supports amd64 Architecture][amd64-shield]
+![Supports armhf Architecture][armhf-shield]
+![Supports armv7 Architecture][armv7-shield]
+![Supports i386 Architecture][i386-shield]
```

Comment:
Maybe I'm seeing this wrong. But doens't Blake Blackshear only provide a docker image for linux/amd64 ?

Reply:
Yes, but this was in the original readme, I just formatted it... I removed the extras.